src/scss/index.scss

```
@@ -1 +1,4 @@
-
+@import "./variables";
```

```suggestion
@import "variables";
```

Same thing, but looks nicer

---

changed